tron/config/config_parse.py

```diff
@@ -70,14 +70,7 @@ def validator(value, config_context):
 
         try:
             valid_context_variable_expr(value, config_context)
-            try:
-                value % context
-                StringFormatter(context).format(value)
-                return value
-            except Exception as e:
-                value % context
-                log.warning("Invalid format string {} at {} {}.\nFall back to % string evaluation".format(e, config_context.path, value))
-                return value
+            return StringFormatter(context).format(value)
```

Comment:
Do the error cases below still apply? i.e. does StringFormatter raise the same exceptions in the same situations?

Reply:
I think they are still needed here. For example, KeyError can be for unknown context variables, while TypeError can be not enough argument (Theoretically possible, but might never happen in our use cases).